paypal/standard/ipn/tests/test_ipn.py

```diff
@@ -207,11 +209,6 @@ def test_reversed_ipn(self):
 
         self.assertGotSignal(payment_was_reversed, False, params, deprecated=True)
 
-    def test_incorrect_receiver_email(self):
-        update = {"receiver_email": "incorrect_email@someotherbusiness.com"}
-        flag_info = "Invalid receiver_email. (incorrect_email@someotherbusiness.com)"
-        self.assertFlagged(update, flag_info)
-
```

Comment:
I wouldn't remove this test yet - the functionality is deprecated, but still needs to work. Instead, it should have an `@override_settings` to set `PAYPAL_RECEIVER_EMAIL` just for this test.